views/template/index.pug

```diff
@@ -0,0 +1,27 @@
+extends layout.pug
+
+block head
+	.container
+		.logo
+			img.head-logo(src="/assets/img/phpbblogo.png")
+			h1 Styleguide
+			img.icon(src="/assets/img/github_icon.png" alt="github")
+		.content
+			p A Styleguide Driven Development System for creating, managaing, customizing, & editing themes for phpBB.
+			p GSoC 2017 project being developed by Sahil Khokhar.
+		.home_btn
+			- var btnType = "info"
+			- var btnSize = "lg"
+			a(href="./demo")
+				button(type="button" class="btn btn-${btnType} btn-${btnSize}") Demo
```

Comment:
Okay, so just CSS around that link? 
Yeah, will shift to base-l in my next push.